depends/README.md

```diff
@@ -43,9 +43,13 @@ The paths are automatically configured and no other options are needed.
 
 ### Install the required dependencies: Ubuntu & Debian
 
+#### Common
+
+    apt install automake bison bsdmainutils cmake curl libtool make patch pkg-config python3 xz-utils
```

Comment:
Right, we could, but https://packages.ubuntu.com/mantic/build-essential is basically gcc, g++ and make. Should we then remove make from this list too, and g++ from the macOS list?

I think I'd actually rather remove `bsdmainutils`, as I really wanted the dependencies listed here to just be what is required to actually build depends, rather than everything needed for Bitcoin Core.

Reply:
> I think I'd actually rather remove bsdmainutils, as I really wanted the dependencies listed here to just be what is required to actually build depends, rather than everything needed for Bitcoin Core.

Works for me.

Reply:
Pushed that now.